tests/test_openclhpp.cpp

```diff
@@ -3035,5 +3040,98 @@ void testDevicePCIBusInfo_KHR()
 #endif
 }
 
+static  cl_int clGetProgramInfo_testProgramGetContext(cl_program program,
+    cl_program_build_info param_name,
+    size_t param_value_size,
+    void *param_value,
+    size_t *param_value_size_ret,
+    int num_calls)
+{
+    (void) num_calls;
+    TEST_ASSERT_EQUAL_PTR(make_program(0), program);
+    TEST_ASSERT_EQUAL_HEX(CL_PROGRAM_CONTEXT, param_name);
+    TEST_ASSERT(param_value == NULL || param_value_size >= sizeof(cl_context));
+    if (param_value_size_ret != NULL)
+        *param_value_size_ret = sizeof(cl_context);
+    if (param_value != NULL)
+        *(cl_context *) param_value = make_context(0);
```

Comment:
perhaps this is an opportunity to experiment with more modern style in new code?
```suggestion
    if (param_value != nullptr)
        *static_cast<cl_context *>(param_value) = make_context(0);
```

Reply:
Done, thanks.